ui/components/ui/color-indicator/color-indicator.stories.js

```diff
@@ -11,30 +11,31 @@ export default {
         type: 'select',
       },
       options: SIZES,
-      defaultValue: SIZES.LG,
     },
     type: {
       control: {
         type: 'select',
       },
       options: ColorIndicator.TYPES,
-      defaultValue: ColorIndicator.TYPES.FILLED,
     },
     color: {
       control: {
         type: 'select',
       },
       options: COLORS,
-      defaultValue: COLORS.PRIMARY_DEFAULT,
     },
     borderColor: {
       control: {
         type: 'select',
       },
       options: { NONE: undefined, ...COLORS },
-      defaultValue: undefined,
     },
   },
+  args: {
+    size: SIZES.LG,
+    type: ColorIndicator.TYPES.FILLED,
+    color: COLORS.PRIMARY_DEFAULT,
+  },
 };
 
 export const DefaultStory = (args) => (
```

Comment:
Color indicator story still works the same as before

### Before
<img width="1440" alt="Screenshot 2022-12-11 at 7 00 11 PM" src="https://user-images.githubusercontent.com/8112138/206958962-f7543ffc-3f90-434c-9bea-04ec442f232b.png">

### After
<img width="1440" alt="Screenshot 2022-12-11 at 6 59 43 PM" src="https://user-images.githubusercontent.com/8112138/206958984-1854943a-2bb9-4836-b645-217b9c21dd58.png">